src/main/pages/che-7/administration-guide/con_che-architectural-components.adoc

```diff
@@ -6,8 +6,10 @@
 As illustrated in the link:{site-baseurl}che-7/high-level-che-architecture[High-level {prod-short} architecture] article, {prod} components are:
 
 * A central workspace controller: an always running service that manages users workspaces
-* Users workspaces: container-based IDEs that the controller stops when the user stops coding
+* Users workspaces: container-based IDEs that the controller stops when the user stops coding.
 
+ifeval::["{project-context}" == "che"]
```

Comment:
now that you have a whitelabel version of the che-high-level image, you can probably remove this if-constraint.